dashboard/app/models/course_offering.rb

```diff
@@ -19,20 +19,22 @@
 #  school_subject                   :string(255)
 #  device_compatibility             :string(255)
 #  description                      :string(255)
-#  self_paced_professional_learning :string(255)
 #  professional_learning_program    :string(255)
 #  video                            :string(255)
 #  published_date                   :datetime
+#  self_paced_pl_course_offering_id :integer
 #
 # Indexes
 #
+#  fk_rails_8283d50b87            (self_paced_pl_course_offering_id)
```

Comment:
Wow! TIL about automatically-generated foreign key names https://guides.rubyonrails.org/v6.0/active_record_migrations.html#foreign-keys

Reply:
I'm worried that foreign keys might trip us up, since we sometimes take liberties with how we delete and recreate data during seeding. If you are ok with it @juanmanzojr , I would suggest simply not using foreign keys for this model (and any other curriculum-related objects), however I'm very interested in differing opinions here if you think it is important to do.

Reply:
to clarify, I think the index is important, but I am worried we will run into foreign key constraints when deleting objects. I haven't dug into your drone output, but it is possible this could be leading to some of the test failures.

Reply:
I had a similar concern, Dave, but I think CourseOffering is okay to use foreign keys (it [uses](https://github.com/code-dot-org/code-dot-org/blob/44784aae5501776ae67433056a30650f92d790cd/dashboard/app/models/course_offering.rb#L373) `find_or_initialize_by`). That said, I had forgotten that we do take liberties with deleting and recreating in other parts of our seeding and agree it might be wise to remove the foreign key, in part for consistency